### PR TITLE
Change notification priority for new talk rooms

### PR DESCRIPTION
**Pull request by oliveiraigorm**
### Changes
Change notification priority for new talk rooms, so all new messages will be notified.

### Testing 

1. Book a new session via the appointments widget. 
2. Confirm it so a talk room is created. 
3. Check the notifications under the room settings and it should be set to All messages

### CODE REVIEW

File: lib/Backend/TalkIntegration.php [hunk 1]
```diff
@@ -87,6 +89,10 @@ function createRoomForEvent(string $attendeeName, $dateTime, string $userId): st
 
         $roomToken = $room->getToken();
 
+        $ps = \OC::$server->get(\OCA\Talk\Service\ParticipantService::class);
```
Comment (mwalbeck) [inline lib/Backend/TalkIntegration.php]: Why are you working with 2 different instances of participantService?

File: lib/Backend/TalkIntegration.php [hunk 2]
```diff
@@ -26,6 +27,7 @@ public function __construct($tlk, $utils) {
         $this->utils = $utils;
         $this->tlk = $tlk;
         $this->config = \OC::$server->get(\OCP\IConfig::class);
+        $this->participantService = \OC::$server->get(\OCA\Talk\Service\ParticipantService::class);
```
Comment (mwalbeck) [inline lib/Backend/TalkIntegration.php:30]: Why isn't dependency injection being used?

**Review by mwalbeck — CHANGES_REQUESTED** [review]

File: lib/Backend/TalkIntegration.php (hunk 2 above)
Comment (oliveiraigorm) [reply lib/Backend/TalkIntegration.php:30]: The class is being initialized by BackendUtils and initializing the variable that way avoids the need to change the other file too.

Comment (mwalbeck) [reply lib/Backend/TalkIntegration.php:30]: Fair

**Review by mwalbeck — APPROVED** [review]

**Pull request merged**